### ISSUE TITLE
Pop OS (Ubuntu) 22.04 Video blackscreen after sleep

### ISSUE THREAD

**Issue by xdubx** [jeffshee/hidamari#130]
If i set my computer in the sleep mode. After i login back the program stays open. But the video does not play anymore.

**Comment by Austint30** [comment]
Similar issue here, but the video is frozen until I kill Hidamari and start it again. This is when I am streaming a video from Youtube.

**Comment by LexGear** [comment]
Yep, after sleep one screen goes black, the other two freeze. This is on Zorin 17

**Comment by mignochrono** [comment]
issue also happens on 
OS:Ubuntu 25.04 x86_64 
Kernel: 6.14.0-22-generic
DE: GNOME 48.0 

is there maybe a way you can script the detection of the sleep mode and kill and relaunch the program?

EDIT:
so after looking on the gnome extension manager it says something about 
Extension hanabi-extension@jeffshee.github.io: TypeError: dockManager is null
and chatgpt(i know ,but im new at linux) says its because hanabi tries to start before the dock service is ready. this makes sense because if i press alt+f2 and type "r" to restart gnome , the extension works again. so maybe it should delay the startup after sleep? but im not knoledgeable enough on linux (or any os for that matter) to do that?
